docs/04-tables.md

```diff
@@ -2,8 +2,8 @@ Tables
 ======
 
 A Sleeper instance contains one or more tables. Each table has four important
-properties: a name, and a schema for storing data for that
-table, and its own state store for storing metadata about the table. 
+properties: a name, a schema for storing data for that table, a state store for 
+storing metadata about the table, and a flag to denote whether the table is online or not. 
 All other resources for the instance, such as the S3 bucket used for storing data 
 in a table, ECS clusters and lambda functions are shared across all the tables.
```

Comment:
It seems like this suggests there are some resources that are not shared across all the tables, which is not correct.